osu.ElasticIndexer/Score.cs

```diff
@@ -69,42 +66,34 @@ public string data
 
         public bool preserve { get; set; }
 
-        [Computed]
         [Number(NumberType.Integer)]
-        public int total_score => (int)scoreData.TotalScore; // scoreData.TotalScore should never exceed int.MaxValue at the point of storage.
+        public int total_score { get; set; } // total score should never exceed int.MaxValue at the point of storage.
 
-        [Computed]
         [Number(NumberType.Integer)]
-        public int legacy_total_score => scoreData.LegacyTotalScore ?? 0;
+        public int legacy_total_score { get; set; }
 
-        [Computed]
         [Number(NumberType.Float)]
-        public double accuracy => scoreData.Accuracy;
+        public double accuracy { get; set; }
 
-        [Computed]
         [Number(NumberType.Integer)]
-        public int max_combo => scoreData.MaxCombo;
+        public int max_combo { get; set; }
 
-        [Computed]
         [Keyword]
-        public string rank => scoreData.Rank.ToString();
+        public string rank { get; set; } = string.Empty;
 
         [Ignore]
         public int user_warnings { get; set; }
 
-        [Computed]
         [Keyword]
-        public List<string> mods => scoreData.Mods.Select(mod => mod.Acronym).ToList();
+        public List<string> mods { get; set; } = null!;
 
-        [Computed]
         [Keyword]
         public string? country_code { get; set; }
 
-        [Computed]
         [Boolean]
-        public bool is_legacy => scoreData.BuildID == null;
+        public bool is_legacy => legacy_score_id != null;
 
-        public ScoreData scoreData = new ScoreData();
+        public ulong? legacy_score_id { get; set; }
```

Comment:
Is this just for `is_legacy` and not to be indexed?

Reply:
yeah, that's the case for now.

Reply:
It doesn't really affect the current usage, but fields that aren't going to be indexed or stored should have `[Ignore, JsonIgnore]` attributes to skip serialization and mapping.